downloader/src/main/kotlin/vcs/GitWorkingTree.kt

```diff
@@ -39,6 +41,19 @@ open class GitWorkingTree(workingDir: File, private val gitBase: GitBase) : Work
         return File(dotGitDir, "shallow").isFile
     }
 
+    override fun getNested(): Map<String, VcsInfo> {
+        val root = getRootPath()
+
+        val paths = gitBase.run(root, "submodule", "status", "--recursive").stdout.lines()
+                .filter { it.isNotBlank() }
+                .map { it.removePrefix("+").removePrefix("-").trim().split(" ")[1] }
+
+        return paths.associate { path ->
+            val workingTree = VersionControlSystem.forDirectory(File(root, path))
```

Comment:
I began to slightly prefer `root.resolve(path)`.

Reply:
Done.